src/modules/others/led.cpp

```diff
@@ -1,56 +1,110 @@
 #ifdef HAS_RGB_LED
-// By @IncursioHack - github.com/IncursioHack
-#include <LiteLED.h>
+#include <FastLED.h>
 #include "core/display.h"
 #include "core/globals.h"
 #include "led_control.h"
 
-// Escolha o tipo de LED na lista abaixo.
-// Comente todos menos um LED_TYPE.
-// #define LED_TYPE        LED_STRIP_WS2812
-#define LED_TYPE        LED_STRIP_SK6812
-// #define LED_STRIP_APA106
-// #define LED_STRIP_SM16703
+#ifdef T_EMBED_1101
```

Comment:
Wouldn't it be better to use the LED_TYPE as condition instead of T-EMBED? ( I'm trying to remove as much Device related MACROS from the ./src/ files and make it as generic as possible)

Otherwise we would need to add more devices in here when we could be declaring this MACRO on platformio.ini..

I would suggest to use like:
```
#if LED_TYPE==WS2812B
   #define LED_ORDER GRB
   #define LED_TYPE_IS_RGBW 0
   #define LED_COUNT 8
#elif LED_TYPE==SK6812
   #define LED_ORDER RGB
   #define LED_TYPE_IS_RGBW 1
   #define LED_COUNT 1
#endif
```

and in cardputer/platformio.ini add `-DLED_TYPE=SK6812` after `-DHAS_RGB_LED=1`
and in t-embed/platformio.ini add `-DLED_TYPE=WS2812B`

if other LED models compatible with FastLED come, we can just make the proper setup in the platformio